src/sentry/static/sentry/app/components/organizations/globalSelectionHeader/globalSelectionHeader.jsx

```diff
@@ -37,6 +40,10 @@ class GlobalSelectionHeader extends React.Component {
      * List of projects to display in project selector
      */
     projects: PropTypes.arrayOf(SentryTypes.Project),
+    /**
+     * If a forced project is passed, selection is disabled
+     */
+    forceProject: SentryTypes.Project,
```

Comment:
Since you use `lock` prop name elsewhere, maybe `lockProject` or `lockToProject` would be more consistent?